package/setup.py

```diff
@@ -286,27 +314,27 @@ def detect_openmp():
                     'MDAnalysis.visualization'],
           package_dir={'MDAnalysis': 'MDAnalysis'},
           ext_package='MDAnalysis',
-          ext_modules=extensions,
+          ext_modules=extensions(config),
           classifiers=CLASSIFIERS,
           long_description=LONG_DESCRIPTION,
           cmdclass=cmdclass,
           # all standard requirements are available through PyPi and
           # typically can be installed without difficulties through setuptools
           install_requires=[
-              'numpy>=1.5.0',  # currently not useful because without numpy we don't get here
-              'biopython>=1.59',  # required for standard PDB reader and sequence alignment
-              'networkx>=1.0',  # LeafletFinder
-              'GridDataFormats>=0.2.2',  # volumes and densities
+              'biopython>=1.59',
+              'networkx>=1.0',
+              'GridDataFormats>=0.2.2',
           ],
           # extras can be difficult to install through setuptools and/or
           # you might prefer to use the version available through your
           # packaging system
           extras_require={
-              'AMBER': ['netCDF4>=1.0'],  # for AMBER netcdf, also needs HDF5 and netcdf-4
+              'AMBER': ['netCDF4>=1.0'],  # for AMBER netcdf, also needs HDF5
+                                          # and netcdf-4
               'analysis': [
                   'matplotlib',
-                  'scipy',  # sparse contact matrix
```

Comment:
At least with ezinstall there was a way to specify extras using something like `ez_install package[analysis]` in order to trigger the extra "analysis". But no-one ever used this as far as I can tell. However, if it doesn't do anything, then why even keeping `extras_requires`?

Additionally, we should actually have `'netCDF4>=1.0'` in `requires` (at least as important as `scipy`) and drop the AMBER extra.